klasses/api.py

```diff
@@ -28,58 +24,3 @@ def payable_bootcamp_run_keys(user):
                     Q(personal_prices__user=user))
             .values_list("run_key", flat=True)
     )
```

Comment:
This function was written as a temporary replacement for the removed `BootcampAdmissionClient`. But like the other functions moved to `ecommerce/api.py` in this PR, it is ecommerce-related and is now the only function left here.  Should it be moved as well so this file can be deleted?

Reply:
Yes, just made the change